dashboard/test/ui/features/footer.feature

```diff
@@ -239,7 +239,7 @@ Feature: Checking the footer appearance
 
     # Additional wait to let scroll position settle and possibly have the
     # pin-to-home-screen popup go away
-    And I wait for 10 seconds
+    And I wait for 20 seconds
```

Comment:
You can definitely watch the ipad test run and see that it takes a while for things to happen in the test environment.

I don't have the expertise to know what to wait to see, if someone who developed the code can improve the precision of the wait that would be great.